restcomm/restcomm.http/src/main/java/org/restcomm/connect/http/ConferencesEndpoint.java

```diff
@@ -194,4 +202,75 @@ protected Response getConferences(final String accountSid, UriInfo info, MediaTy
         }
     }
 
+    protected Response updateConference(String accountSid, String sid, MultivaluedMap<String, String> data,
+            MediaType responseType) {
+        logger.info(String.format("updateConference accountsid: %s conferenceSid: %s", accountSid, sid));
+        Account account = daoManager.getAccountsDao().getAccount(accountSid);
+        Account effectiveAccount = userIdentityContext.getEffectiveAccount();
+        try {
+            secure(account, "RestComm:Modify:Conferences");
+        } catch (final AuthorizationException exception) {
+            return status(UNAUTHORIZED).build();
+        }
+        final ConferenceDetailRecordsDao dao = daoManager.getConferenceDetailRecordsDao();
+        ConferenceDetailRecord cdr = dao.getConferenceDetailRecord(new Sid(sid));
+        if (cdr == null) {
+            return status(NOT_FOUND).build();
+        } else {
+            //allow super admin to perform update on any conference - https://telestax.atlassian.net/browse/RESTCOMM-1171
+            if (!isSuperAdmin()) {
+                try {
+                    secure(account, cdr.getAccountSid(), SecuredType.SECURED_STANDARD);
```

Comment:
@maria-farooq can you elaborate on that? Why we use secure again here?

Reply:
conference termination request can be initiated by owner of the conference as well hence we need to make sure that resource is being accessed by proper account